src/Inspector/Inspector.tsx

```diff
@@ -103,6 +103,11 @@ export const Inspector: React.FC<InspectorProps> = (props) => {
     overlay.setRemoveCallback(stopCallback)
 
     overlayRef.current = overlay
+
+    const { scrollLeft, scrollTop } = document.documentElement
+    const element = document.elementFromPoint(scrollLeft, scrollTop)
+
+    handleHoverElement(element as HTMLElement)
```

Comment:
I try it online, but feels the interaction is weird.

Maybe I want is inspect at mouse point at start, so we need listen mouse move and recode point at this Inspector component mounted.

Reply:
My opinion is that, when hot keys are pressed, if user doesn't move mouse, the overlay doesn't trigger, there's no visual indicator as well. Having the overlay appearing upon hotkeys goes the developers a sense that there's something happening. My use-case is to create a cra template and pre install this tool, but newer developers might get a bit lost